packages/wasm/wasm.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "wasm"
+version: "0.1"
```

Comment:
Could you please remove the `name` and `version` fields here? They are redundant and easily get out of sync from the same metadata in the path of the package.